optuna/samplers/_cmaes.py

```diff
@@ -499,6 +558,13 @@ def _split_optimizer_str(optimizer_str: str) -> Dict[str, str]:
     return attrs
 
 
+def _is_compatible_search_space(
+    trans: _SearchSpaceTransform, search_space: Dict[str, BaseDistribution]
+) -> bool:
+    union_size = len(set(trans._search_space.keys()).intersection(search_space.keys()))
```

Comment:
A nit, but the righthand side takes `intersection`, and I think the variable name should be `intersection_size`.

```suggestion
    intersection_size = len(set(trans._search_space.keys()).intersection(search_space.keys()))
```

Reply:
Oh, thanks!